dashboards-observability/public/components/common/side_nav.tsx

```diff
@@ -32,7 +32,7 @@ export function ObservabilitySideBar(props: { children: React.ReactNode }) {
     // Default page is Events Analytics
     // But it is kept as second option in side nav
     if (hash === '#/') {
-      items[0].items[1].isSelected = true;
+      items[0].items[2].isSelected = true;
```

Comment:
Just want to verify that here that the index should be changed to `2` - I see in the comment above that the second option in side nav should be kept

Reply:
It needs to be changed to 2 once Application Analytics is added as an option above Event Analytics. This might not be necessary if the App Analytics option is below.